bin.js

```diff
@@ -100,7 +99,7 @@ function create (dir, argv) {
     },
     function (done) {
       var pkgs = [
-        'bankai@next',
+        'bankai',
```

Comment:
is bankai not on `rc5`, etc anymore, or will this resolve to that? (might be a silly question, sorry if it is)

Reply:
Oh yeah, so `npm install bankai` installs the latest RC out of the box right now. We messed up somewhere, but ah well — at least this code does what it's supposed to do this way (:

Reply:
oh cool! i was asking cause couple of weeks ago it would just install 9.0.0, but i guess something got fixed!